study.md

```diff
@@ -38,7 +38,7 @@ manipulates it by sorting it and then reversing it in place (i.e., modifying the
 reference to the original).
 
 ```ruby
-# your answer here
+[23, 56, 3, 7].sort.reverse
 ```
```

Comment:
This will not actually modify this in place.. What operator do we need to do that?